### DISCUSSION TITLE
How many game records (incl. Gold, Jade, and Throne) are required for training in order to get a "decent" model?

### DISCUSSION THREAD

**Discussion by FYNIXqwq** [Cryolite/kanachan#57] [Q&A] [answered]
I'm now slowly accumulating game records.

**Comment by shinkuan** [comment]
For me, 100Mil samples (not games) is enough to train a BERT model.
![download](https://github.com/Cryolite/kanachan/assets/35415788/dbc0003d-0df7-401b-9558-8e74157dd17d)

**Reply by FYNIXqwq** [reply]
Thanks. Then how many Hanchans are approximately equivalent to 100M samples?

**Reply by Cryolite** [reply, marked as answer]
On average, you can extract just under 150 samples per Hanchan. Therefore, by simple calculation, it would take about 7,000 Hanchan to gather 1M samples. However, since the samples extracted from a single Hanchan are highly correlated with each other, it would be better to collect samples from as many Hanchans as possible and then randomly sample the training examples from them.
